packages/amplify-graphql-api-construct/API.md

```diff
@@ -125,6 +125,7 @@ export interface AmplifyGraphqlApiProps {
     readonly apiName?: string;
     readonly authorizationModes: AuthorizationModes;
     readonly conflictResolution?: ConflictResolution;
```

Comment:
Shouldn't this be deprecated now? Maybe API.md doesn't support the deprecation flag though

Reply:
It looks like this is because I included a comma after the `@deprecated`. There are a few other instances of this for unrelated properties. I'll create a separate pr to fix this.

Reply:
I've fixed two `@deprecated` tags added in this PR.

I looked into fixing all other issues using `jsdoc/check-tag-names` eslint rule. There are many more occurrences than I thought. I've created a backlog item to fix this later.